### DISCUSSION TITLE
First Month - WOW

### DISCUSSION THREAD

**Discussion by IanBlair07** [mcguirepr89/BirdNET-Pi#1201] [Show and tell]
My Birdnet-Pi has been running for one month. It's on the edge of my 5-acres of open woodland in central Indiana, which is contiguous with around 100 acres of woods. I am amazed to record 86 species of birds in the first month.  Thanks for everyone who supports this effort with their time and talent. Attached is a summary of the species.
[Species Report 051424.pdf] (https://github.com/mcguirepr89/BirdNET-Pi/files/15314443/Species.Report.051424.pdf)
Ian Blair

**Comment by jpbell** [comment]
This is an impressive list of birds, but I am also curious how you created it?

**Reply by IanBlair07** [reply]
Hi JP,
I use MS Excel to do the heavy lifting. I am attaching a sample file with data from location for 6/1/24 to 6/4/24. Also attached are some brief instructions. So of course you could substitute your data. Let me know how it turns out.
Best wishes,
Ian Blair

[June 2024.xlsx](https://github.com/user-attachments/files/15597945/June.2024.xlsx)
[Creating Species Report.pdf](https://github.com/user-attachments/files/15597947/Creating.Species.Report.pdf)

**Reply by jpbell** [reply]
Great! I'll give this a shot later tonight and post the results.

**Reply by jpbell** [reply]
This is what I ended up with. Not as long as yours but I have only had it up and running for about a week... Thanks for throwing this together!
[Bird Report 1.pdf](https://github.com/user-attachments/files/15603368/Bird.Report.1.pdf)

**Reply by IanBlair07** [reply]
Hey, great results. I love to see that Barred Owl with 0.83 confidence. I had to look up the Black-billed Cuckoo since I wasn't familiar with it. Best wishes. Ian
